### PR TITLE
Add checklist to update maintainers team

### PR DESCRIPTION
**Pull request by edmundmiller**
- **chore: Add Slack Message from @jfy133**
- **Try to use fancy checklists**

@netlify /docs/checklists/maintainers_team_updates

### CODE REVIEW

**Comment by jfy133** [comment]
@edmundmiller I don't seem to be able to find the page (either by clicking through the website or guessing the URL), any ideas?

**Comment by jfy133** [comment]
Oh, wait found it!

File: src/content/docs/checklists/maintainers_team_updates.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Maintainers Team Updates
+subtitle: A step-by-step guide for changing of the guard
+markdownPlugin: checklist
+---
+
```
Comment (jfy133) [inline src/content/docs/checklists/maintainers_team_updates.mdx:6]:
```suggestion
Roughly once a year we review the maintainers team to ensure we have sufficient capacity to keep up with the community activity. 

The following steps are what the maintainers team leads need to do during the review. 

```

File: src/content/docs/checklists/maintainers_team_updates.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+1. [ ] Ask who wants to step down
+2. [ ] Ask for nominations in maintainers meeting, discuss and approve
+3. [ ] Ask nominees
+4. [ ] Once accepted, add to [#team-maintainers](https://nfcore.slack.com/archives/C043UU89KKQ) (and remove alumni/extras)
+5. [ ] Once accepted, make PRs and merge in:
```
Comment (jfy133) [inline src/content/docs/checklists/maintainers_team_updates.mdx:11]:
```suggestion
5. [ ] Once accepted, make PRs to add/move to alumni members and merge in:
```

**Review by jfy133 — APPROVED** [review]
LGTM but I sort of wonder if it makes sense to have this page exposed in the docs overview page... It's only basically for 3 people +maybe the maintainers team in an an emergency (technically)

**Comment by mashehu** [comment]
> LGTM but I sort of wonder if it makes sense to have this page exposed in the docs overview page... It's only basically for 3 people +maybe the maintainers team in an an emergency (technically)

Yeah, had the same feeling...

**Comment by jfy133** [comment]
Can we leave the file in the same place but not index it in the docs overview section @mashehu ? Like 'hidden: true' in the header or something?

**Comment by mashehu** [comment]
We don't have that option, not sure if we really want to have hidden files...

**Comment by jfy133** [comment]
Hidden only from being displayed in the overview, not that the page isn't accessible at all (if you see what I mean).

**Comment by edmundmiller** [comment]
> Hidden only from being displayed in the overview, not that the page isn't accessible at all (in case I wasn't clear)

Yeah, I agree with this. I don't think there's anything sensitive in there, but it would just be clutter that most people won't ever need.

**Comment by jfy133** [comment]
Whelp forgot about this, what about  a section under 'Guidelines' called 'Governance', allows transprency but not something someone would normally click on

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+---
+title: Core Checklists
+subtitle: Step-by-step guides for core team tasks
+markdownPlugin: checklist
+---
+
+## New pipeline onboarding
+
+This documents the steps that need to be performed when a new pipeline is accepted via #new-pipelines
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:9]:
```suggestion
This document outlines the steps to be performed when a new pipeline is accepted via #new-pipelines channel
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+1. [ ] Core: Check proposal has received approval on #new-pipelines from either:
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:11]:
```suggestion
1. [ ] Core: Check proposal has received approval in #new-pipelines from either:
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    - at least 2 members of core team
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:12]:
```suggestion
    - at least two members of the core team
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    - at least 1 member of core team and 1 member of maintainers team
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:13]:
```suggestion
    - at least one member of the core team and one member of the maintainers team.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+2. [ ] Core: Check name follows [pipeline naming guidelines](/docs/guidelines/pipelines/workflow_name)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:14]:
```suggestion
2. [ ] Core: Verify that the pipeline name follows the [pipeline naming guidelines](/docs/guidelines/pipelines/workflow_name).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+3. [ ] Core: Make slack channel with approved name
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:15]:
```suggestion
3. [ ] Core: Create a slack channel with approved pipeline name.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+4. [ ] Core: Ask developer to join #pipeline-maintainers
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:16]:
```suggestion
4. [ ] Core: Ask the developers to join #pipeline-maintainers channel.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+5. [ ] Core/Dev: Make git repository with nf-core create and push to GitHub
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:17]:
```suggestion
5. [ ] Core/Dev: Create the git repository using `nf-core create` and push it to GitHub.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+6. [ ] Core/Dev: Transfer repository to nf-core GitHub organisation and delete repository on local machine
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:18]:
```suggestion
6. [ ] Core/Dev: Transfer the repository to nf-core GitHub organisation and delete the repository on local machine.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+7. [ ] Core/Dev: Sync the nf-co.re pipelines page by manually triggering the GitHub action (website repo > actions > build json files and md-cache > Run workflow > from main)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:19]:
```suggestion
7. [ ] Core/Dev: Sync the nf-co.re pipelines page by manually triggering the GitHub action (website repo -> actions -> build json files and md-cache -> Run workflow -> from main).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
+## Uploading test data to s3 bucket
+
+Uploading full-test AWS files for a pipeline to S3 bucket information:
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:23]:
```suggestion
Instructions for uploading full-test AWS files for a pipeline to a S3 bucket:
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+
+-   [ ] Check Valid test data that can be uploaded:
+    - [ ] Small input data that maybe flaky when downloaded from other repositories
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:26]:
```suggestion
    - [ ] Small input data that maybe unreliable when downloaded from other repositories
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+            e.g. Max of 10GB for the total across all files
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:27]:
```suggestion
            (e.g., limit total file size to a maximum of 10GB across all files)
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+    - [ ] Very large files that cannot be reduced or subset
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:28]:
```suggestion
    - [ ] Very large files that cannot be reduced or subsetted.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+-   [ ] When creating the pipeline directory in the bucket, use the 'Amazon S3-managed keys (SSE-S3) option
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:29]:
```suggestion
-   [ ] When creating the pipeline directory in the S3 bucket, select the 'Amazon S3-managed keys (SSE-S3)` option.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+
+## Activating Zenodo archiving of a new pipeline
+
+Instructions for setting up Zenodo DOIs.
+Anyone who is in the nf-core organization can do this!
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:34]:
```suggestion
Instructions for setting up Zenodo DOIs. Anyone in the nf-core organization can do this!
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+
+Before release:
+
+1. [ ] Sign up with Zenodo and log in
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:38]:
```suggestion
1. [ ] Sign up on Zenodo and log in.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
+2. [ ] Make sure you have connected your github (Settings -> GitHub -> Connect Account)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:39]:
```suggestion
2. [ ] Connect your Github account (Go to Settings -> GitHub -> Connect Account).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 21]
```diff
@@ -0,0 +1,105 @@
...
+3. [ ] Once connected, go back to Settings -> GitHub
+4. [ ] Flip switch against all nf-core _pipeline_ repositories in the list of repositories to 'On' (including the one that you want to assign)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:41]:
```suggestion
4. [ ] Enable Zenodo archiving for all nf-core pipeline repositories by toggling the switch to `On` (including the one you want to assign).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 22]
```diff
@@ -0,0 +1,105 @@
...
+    - Might as well do this for all (active) pipelines you see, as DOIs won't be assigned until release anyway
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:42]:
```suggestion
    - It’s a good idea to enable this for all active pipelines, as DOIs will only be assigned upon release.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 23]
```diff
@@ -0,0 +1,105 @@
...
+    - Some will not activate (e.g. nf-core/exoseq) as they are archived!
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:43]:
```suggestion
    - Note: Some repositories (e.g. nf-core/exoseq) may not activate if they are archived.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 24]
```diff
@@ -0,0 +1,105 @@
...
+5. [ ] Tell pipeline developer to make a release
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:44]:
```suggestion
5. [ ] Inform the pipeline developers to make a release.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 25]
```diff
@@ -0,0 +1,105 @@
...
+
+Post release:
+
+1. [ ] Go to Zenodo and find the repository of interest on the list repositories under Settings -> Github
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:48]:
```suggestion
1. [ ] Go to Zenodo and find the relevant repository in Settings -> Github.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 26]
```diff
@@ -0,0 +1,105 @@
...
+2. [ ] Click through to the release's Zenodo 'record' page
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:49]:
```suggestion
2. [ ] Click through to the Zenodo 'record' page for the release.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 27]
```diff
@@ -0,0 +1,105 @@
...
+3. [ ] Scroll down and look on the right-hand side of the page for the 'Communities' box, submit the record to the nf-core community
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:50]:
```suggestion
3. [ ] On the record page, scroll down and look on the right-hand side and find the `Communities` box. Submit the record to the nf-core community.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 28]
```diff
@@ -0,0 +1,105 @@
...
+4. [ ] Scroll down and again look on the right-hand side of the page for the 'Versions' box
+5. [ ] Copy the 'cite all versions by using the DOI ' DOI (_not_ the specific release!)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:52]:
```suggestion
4. [ ] Scroll down and again on the right-hand side of the page for the `Versions` box, copy the DOI for 'cite all versions by using the DOI ' (_not_ the specific release DOI).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 29]
```diff
@@ -0,0 +1,105 @@
...
+6. [ ] On the master branch of the pipeline's repository, edit the following files:
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:53]:
```suggestion
5. [ ] Update the following files on the master branch of the pipeline repository:
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 30]
```diff
@@ -0,0 +1,105 @@
...
+    - **README.md**: Zenodo Badge at the top, and 'If you use this pipeline cite:' at the bottom (make sure to uncomment this section too)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:54]:
```suggestion
    - **README.md**: Add the Zenodo Badge at the top and update the 'If you use this pipeline cite:' section at the bottom (make sure to uncomment this section too).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 31]
```diff
@@ -0,0 +1,105 @@
...
+    - **Nextflow.config**: manifest's DOI entry
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:55]:
```suggestion
    - **Nextflow.config**: Update the manifest block to include the DOI.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 32]
```diff
@@ -0,0 +1,105 @@
...
+    - Commit with something like 'Add Zenodo ID after first release'
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:56]:
```suggestion
    - Commit these changes with a message like 'Add Zenodo ID after first release'.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 33]
```diff
@@ -0,0 +1,105 @@
...
+
+## Adding new community member to the GitHub organisation
+
+Anyone can request to join the nf-core GitHub organisation via #github-invitations.
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:60]:
```suggestion
Anyone can request to join the nf-core GitHub organisation via #github-invitations channel.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 34]
```diff
@@ -0,0 +1,105 @@
...
+
+1.  [ ] Check they have a reasonable to join (clearly not spam)
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:62]:
```suggestion
1.  [ ] Verify the request is reasonable (i.e., clearly not spam).
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 35]
```diff
@@ -0,0 +1,105 @@
...
+2.  [ ] Invite via https://github.com/orgs/nf-core/teams
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:63]:
```suggestion
2.  [ ] Send an invitation via [nf-core github teams](https://github.com/orgs/nf-core/teams)
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 36]
```diff
@@ -0,0 +1,105 @@
...
+3.  [ ] Make sure to do team 'contributors' https://github.com/orgs/nf-core/teams/contributors
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:64]:
```suggestion
3.  [ ] Add them to the [nf-core contributors team](https://github.com/orgs/nf-core/teams/contributors)
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 37]
```diff
@@ -0,0 +1,105 @@
...
+4.  [ ] Press ✅ on their message in #github-invitations
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:65]:
```suggestion
4.  [ ] Confirm by reacting with ✅ on their message in #github-invitations channel.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 38]
```diff
@@ -0,0 +1,105 @@
...
+
+## Updating online bingo cards
+
+We need to update the [bingo cards](https://nfcore-bingo.web.app/) before each hackathon to ensure they remain up-to-date.
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:69]:
```suggestion
Before each hackathon, we need to update the [bingo cards](https://nfcore-bingo.web.app/) to ensure they remain current.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 39]
```diff
@@ -0,0 +1,105 @@
...
+
+1. [ ] Get the outreach google account credentials
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:71]:
```suggestion
1. [ ] Get the outreach google account credentials from `@core-team`
```

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 40]
```diff
@@ -0,0 +1,104 @@
...
+
+This document outlines the steps to be performed when a new pipeline is accepted via the #new-pipelines channel
+
+1. [ ] Core: Check proposal has received approval in #new-pipelines from either:
+    - at least two members of the core team
+    - at least one member of the core team and one member of the maintainers team.
+2. [ ] Core: Verify that the pipeline name follows the [pipeline naming guidelines](/docs/guidelines/pipelines/workflow_name).
+3. [ ] Core: Create a slack channel with approved pipeline name.
+4. [ ] Core: Ask the developers to join #pipeline-maintainers channel.
+5. [ ] Core/Dev: Create the git repository using `nf-core create` and push it to GitHub.
```
Comment (sateeshperi) [inline src/content/docs/checklists/community_governance/core_team.mdx:17]:
```suggestion
5. [ ] Core/Dev: Create the git repository using `nf-core pipelines create` and push it to GitHub.
```

**Review by sateeshperi — APPROVED** [review]

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 41]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+## Maintainers Team Onboarding
+
+Team Leads:
+
+1. [ ] Add member to [#team-maintainers](https://nfcore.slack.com/archives/C043UU89KKQ)
+2. [ ] Add member to [Github group](https://github.com/orgs/nf-core/teams/maintainers)
+3. [ ] Add member to Slack group
+4. [ ] Add new maintainers to the [Maintainers team google calendar](https://calendar.google.com/calendar/u/0?cid=ZDA3MmVkOGI1NzQ0YWViOWE3ZjBkMDE4YjFjZWE1OGE5M2MwZDZlZTIyMzM1YWE5YzA0NTlmODg0YTZkMDNkZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:14]:
```suggestion
4. [ ] Add member to the [Maintainers team google calendar](https://calendar.google.com/calendar/u/0?cid=ZDA3MmVkOGI1NzQ0YWViOWE3ZjBkMDE4YjFjZWE1OGE5M2MwZDZlZTIyMzM1YWE5YzA0NTlmODg0YTZkMDNkZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 42]
```diff
@@ -0,0 +1,33 @@
+---
+title: Maintainers Checklists
+subtitle: Step-by-step guides for maintainers team tasks
+markdownPlugin: checklist
+---
+
+## Maintainers Team Onboarding
+
+Team Leads:
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:9]:
```suggestion
Team lead:
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 43]
```diff
@@ -0,0 +1,33 @@
...
+
+1. [ ] Add member to [#team-maintainers](https://nfcore.slack.com/archives/C043UU89KKQ)
+2. [ ] Add member to [Github group](https://github.com/orgs/nf-core/teams/maintainers)
+3. [ ] Add member to Slack group
+4. [ ] Add new maintainers to the [Maintainers team google calendar](https://calendar.google.com/calendar/u/0?cid=ZDA3MmVkOGI1NzQ0YWViOWE3ZjBkMDE4YjFjZWE1OGE5M2MwZDZlZTIyMzM1YWE5YzA0NTlmODg0YTZkMDNkZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+
+New Maintainer:
+
+1. [ ] Make a PR to nf-core website adding themselves to the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:18]:
```suggestion
1. [ ] Add self profile to the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 44]
```diff
@@ -0,0 +1,33 @@
...
+
+## Maintainers Team Yearly Maintenance
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:20]:
```suggestion
## Maintainers team membership
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 45]
```diff
@@ -0,0 +1,33 @@
...
+
+:::note
+Maintainers can be added at any point of the year at the nomination of the community.
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:23]:
```suggestion
New maintainers may be nominated by the community and join the team at any time.
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 46]
```diff
@@ -0,0 +1,33 @@
...
+Yearly maintenance is performed to ensure everything is up-to-date particularly in regards to members who step down.
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:24]:
```suggestion
Maintainers maintenance is performed yearly to ensure membership is up-to-date, particularly in regards to members who wish to step down due to new or changing commitments.
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 47]
```diff
@@ -0,0 +1,33 @@
...
+:::
+
+1. [ ] Ask who wants to step down
+2. [ ] Ask for nominations in maintainers meeting, discuss and approve
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:28]:
```suggestion
2. [ ] Ask for nominations, discuss, and approve new members
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 48]
```diff
@@ -0,0 +1,33 @@
...
+3. [ ] Invite nominees to join
+4. [ ] Once accepted, nominee and team leads perform tasks [onboarding](#maintainers-team-onboarding)
+5. [ ] Move members who are stepping down to alumni status on website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:31]:
```suggestion
5. [ ] Move members who step down to alumni status on the website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 49]
```diff
@@ -0,0 +1,33 @@
...
+
+6. [ ] Announce on [#announcements](https://nfcore.slack.com/archives/CE6P95170)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:33]:
```suggestion
6. [ ] Announce new members on [#announcements](https://nfcore.slack.com/archives/CE6P95170)
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 50]
```diff
@@ -0,0 +1,104 @@
+---
+title: Core Checklists
+subtitle: Step-by-step guides for core team tasks
+markdownPlugin: checklist
+---
+
+## New pipeline onboarding
+
+This document outlines the steps to be performed when a new pipeline is accepted via the #new-pipelines channel
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/core_team.mdx:9]:
```suggestion
Instructions for nf-core pipeline approval:
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 51]
```diff
@@ -0,0 +1,33 @@
+---
+title: Maintainers Checklists
+subtitle: Step-by-step guides for maintainers team tasks
+markdownPlugin: checklist
+---
+
+## Maintainers Team Onboarding
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:7]:
```suggestion
## Maintainers team onboarding
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 52]
```diff
@@ -0,0 +1,104 @@
...
+-   [ ] Check Valid test data that can be uploaded:
+    -   [ ] Small input data that maybe unreliable when downloaded from other repositories
+            (e.g., limit total file size to a maximum of 10GB across all files)
+    -   [ ] Very large files that cannot be reduced or subsetted.
+-   [ ] When creating the pipeline directory in the S3 bucket, select the 'Amazon S3-managed keys (SSE-S3)` option.
+
+## Activating Zenodo archiving of a new pipeline
+
+Instructions for setting up Zenodo DOIs.
+It's recommended that a core team member does these steps as it makes it easier for us to transfer to the nf-core Zenodo community.
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/core_team.mdx:34]:
```suggestion

:::note
It's recommended that a core team member transfers the DOI to the nf-core Zenodo community.
:::
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 53]
```diff
@@ -0,0 +1,104 @@
...
+
+Before release:
+
+1. [ ] Sign up on Zenodo and log in.
+2. [ ] Connect your Github account (Go to Settings -> GitHub -> Connect Account).
+3. [ ] Once connected, go back to Settings -> GitHub
+4. [ ] Enable Zenodo archiving for all nf-core pipeline repositories by toggling the switch to `On` (including the one you want to assign).
+    - It’s a good idea to enable this for all active pipelines, as DOIs will only be assigned upon release.
+    - Note: Some repositories (e.g. nf-core/exoseq) may not activate as they are archived.
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/core_team.mdx:43]:
```suggestion
    - Note: Some repositories (e.g., nf-core/exoseq) may not activate as they are archived.
```

File: src/content/docs/checklists/community_governance/core_team.mdx [hunk 54]
```diff
@@ -0,0 +1,104 @@
...
+4.  [ ] Respond to the request on slack by reacting with ✅ on their message in #github-invitations channel (so they get their acceptance instructions).
+
+## Updating online bingo cards
+
+Before each hackathon, we need to update the [bingo cards](https://nfcore-bingo.web.app/) to ensure they remain current.
+
+1. [ ] Get the outreach google account credentials from `@core-team`
+2. [ ] Log into the Firebase Dashboard: https://console.firebase.google.com/
+3. [ ] Click “Realtime Database” on the left
+4. [ ] In the middle click “games” and navigate down the tree until you reach “lexicon”
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/core_team.mdx:73]:
```suggestion
4. [ ] Click “games” and navigate down the tree until you reach “lexicon”
```

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
It looks pretty good to me. A few minor comments to remove excess words.

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 55]
```diff
@@ -0,0 +1,33 @@
...
+Maintainers maintenance is performed yearly to ensure membership is up-to-date, particularly in regards to members who wish to step down due to new or changing commitments.
+:::
+
+1. [ ] Ask who wants to step down
+2. [ ] Ask for nominations, discuss, and approve new members
+3. [ ] Invite nominees to join
+4. [ ] Once accepted, nominee and team leads perform tasks [onboarding](#maintainers-team-onboarding)
+5. [ ] Move members who step down to alumni status on the website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
+
+6. [ ] Announce new members on [#announcements](https://nfcore.slack.com/archives/CE6P95170)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:33]:
```suggestion
1. [ ] Ask who wants to step down.
2. [ ] Ask for nominations, discuss, and approve new members.
3. [ ] Invite nominees to join.
4. [ ] Once accepted, nominee and team leads perform tasks [onboarding](#maintainers-team-onboarding).
5. [ ] Move members who step down to alumni status on the website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers).

6. [ ] Announce new members on [#announcements](https://nfcore.slack.com/archives/CE6P95170).
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 56]
```diff
@@ -0,0 +1,33 @@
...
+Team lead:
+
+1. [ ] Add member to [#team-maintainers](https://nfcore.slack.com/archives/C043UU89KKQ)
+2. [ ] Add member to [Github group](https://github.com/orgs/nf-core/teams/maintainers)
+3. [ ] Add member to Slack group
+4. [ ] Add member to the [Maintainers team google calendar](https://calendar.google.com/calendar/u/0?cid=ZDA3MmVkOGI1NzQ0YWViOWE3ZjBkMDE4YjFjZWE1OGE5M2MwZDZlZTIyMzM1YWE5YzA0NTlmODg0YTZkMDNkZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+
+New Maintainer:
+
+1. [ ] Add self profile to the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:18]:
```suggestion
1. [ ] Add profile [**About > Governance > Maintainers**](https://nf-co.re/governance#maintainers)
```

File: src/content/docs/checklists/community_governance/maintainers_team.mdx [hunk 57]
```diff
@@ -0,0 +1,33 @@
...
+:::note
+New maintainers may be nominated by the community and join the team at any time.
+Maintainers maintenance is performed yearly to ensure membership is up-to-date, particularly in regards to members who wish to step down due to new or changing commitments.
+:::
+
+1. [ ] Ask who wants to step down
+2. [ ] Ask for nominations, discuss, and approve new members
+3. [ ] Invite nominees to join
+4. [ ] Once accepted, nominee and team leads perform tasks [onboarding](#maintainers-team-onboarding)
+5. [ ] Move members who step down to alumni status on the website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```
Comment (christopher-hakkaart) [inline src/content/docs/checklists/community_governance/maintainers_team.mdx:31]:
```suggestion
5. [ ] Move members who step down to alumni status on the website in the [About > Governance > Maintainers section](https://nf-co.re/governance#maintainers)
```
```suggestion
5. [ ] Move members who step down to alumni status on the website in [**About > Governance > Maintainers**](https://nf-co.re/governance#maintainers)
```

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
I'd like to trend towards some consistent stylistic changes. Content is the same. Please check formatting of the notes I added.

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]

**Review by christopher-hakkaart — APPROVED** [review]

**Pull request merged**